modules/dynamic-provisioning-azure-file-considerations.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// storage/persistent-storage/persistent-storage-azure-file.adoc
+
+[id="azure-file-considerations_{context}"]
+= Considerations when using Azure File
```

Comment:
The line is render as "CONSIDERATIONS WHEN USING AZURE FILE" does not look correct.